.github/workflows/release.yml

```diff
@@ -7,28 +7,36 @@ on:
 
 env:
   SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+  PYTHONUNBUFFERED: 1
   
 jobs:
   sonar_release:
     runs-on: ubuntu-latest
     name: Start release process
     steps:
-      - name: Checkout release action
-        uses: actions/checkout@v2
-        with:
-          repository: SonarSource/gh-action_LT_release
-
       - name: LT release
-        id: lt_release
-        
+        id: lt_release        
         with:
+          publish_to_binaries: true
           distribute: true
           attach_artifacts_to_github_release: true
           run_rules_cov: true
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
for my education @tomverin  - the GH_TOKEN is not used anymore? I see it's not present in the environment anymore

Reply:
actually, I see there's a `RELEASE_GITHUB_TOKEN` now, so nevermind 👍